rllib/tests/test_rllib_train_and_evaluate.py

```diff
@@ -298,11 +298,12 @@ def test_json_run(self):
     def test_python_run(self):
         assert os.popen(
             f"python {rllib_dir}/scripts.py train file tuned_examples/simple_q/"
-            f"cartpole_simpleq_test.py --type=python"
+            f"cartpole_simpleq_test.py --type=python "
```

Comment:
See my comment above on auto-file-type detection. Seems too much burden on user to say:
```
--file [abc].py --type=python
```
when it's already quite obvious that it's a python file :)

Reply:
good point, can be done.

Reply:
done